src/cluster.c

```diff
@@ -2823,64 +2792,76 @@ valkeyClusterGetValkeyAsyncContext(valkeyClusterAsyncContext *acc,
     return ac;
 }
 
-valkeyClusterAsyncContext *valkeyClusterAsyncContextInit(void) {
+valkeyClusterAsyncContext *valkeyClusterAsyncContextInit(const valkeyClusterOptions *options) {
```

Comment:
Is there any point in supporting init without connect?

valkeyClusterAsyncConnectWithOptions would be enough, wouldn't it?